diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/FileSize.kt

```diff
@@ -25,6 +25,7 @@ class FileSize : Rule("file-size") {
     private var fileName: String? = null
     private var isFixMode: Boolean = false
 
+    @Suppress("NestedBlockDepth")
```

Comment:
Did you by accident merge with your local copy of master branch? This line is already removed in remote.